python/tvm/tir/analysis/analysis.py

```diff
@@ -16,9 +16,14 @@
 # under the License.
 """Wrapping existing analysis utils."""
 # pylint: disable=invalid-name
-
+from __future__ import annotations
```

Comment:
I hit an issue a while ago that the current TVM version probably didn't support this :-(